vespamalloc/src/vespamalloc/malloc/memblockboundscheck.h

```diff
@@ -45,6 +59,12 @@ class MemBlockBoundsCheckBaseTBase : public CommonT<5>
     void verifyFill() const __attribute__((noinline));
 
     void setSize(size_t sz) { static_cast<uint64_t *>(_ptr)[0] = sz; }
+    static size_t preambleOverhead(std::align_val_t alignment) {
+        return std::max(4*sizeof(unsigned), size_t(alignment));
+    }
+    static size_t preambleOverhead() {
+        return 4*sizeof(unsigned);
+    }
```

Comment:
Both these could be `constexpr` (though the compiler will do this in practice anyway)

Reply:
Done